crates/db_schema/src/impls/instance.rs

```diff
@@ -94,11 +94,15 @@ impl Instance {
       .await
   }
 
-  #[cfg(test)]
+  /// Only for use in tests
   pub async fn delete_all(pool: &mut DbPool<'_>) -> Result<usize, Error> {
```

Comment:
Had to remove this because items marked as `#[cfg(test)]` are only available within the same crate.